src/validators/name.tsx

```diff
@@ -0,0 +1,13 @@
+export function validateName(name: string) {
+  if (!name) {
+    return "Name is required.";
+  }
+
+  if (name.length < 3) {
+    return "Name must be at least 3 characters long.";
+  }
+
+  if (name.length > 75) {
+    return "Name must be at most 75 characters long.";
+  }
+}
```

Comment:
passar o min e max por parâmetro?
```suggestion
interface ValidateNameOptions {
  minLength: string;
  maxLength: string;
}
export function validateName(name: string, { minLength, maxLength }: ValidateNameOptions) {
  if (!name) {
    return "Name is required.";
  }

  if (name.length < minLength) {
    return `Name must be at least ${minLength} characters long.`;
  }

  if (name.length > maxLength) {
    return `Name must be at most ${maxLength} characters long.`;
  }
}
```

Reply:
Boa!

Valeu Beto! Corrigido 👍🏻